louise.py

```diff
@@ -0,0 +1,24 @@
+balance = 2000
+min_wd = 50
+max_wd = 1000
+
+def ATM(withdrawal: int):
+    choice = input("Do you want to withdraw money? Type 'yes' or 'no': ")
+    if choice == "no":
+        print("goodbye")
+        return
+    elif choice == 'yes':
+        valid_amount = False
+        while not valid_amount:
+            amount = int(input("How much? "))
```

Comment:
this is logically correct, but it can be catastrophic in future, especially when the user inputs something that is not an integer, say "adam"